torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -476,6 +476,10 @@ void IrPrinter::handle(const GatherOp* op) {
   os_ << "} )\n";
 }
 
+void IrPrinter::handle(const ViewDtypeOp* top) {
+  indent() << top->out() << " = view.dtype( " << top->in() << " )\n";
```

Comment:
`top->out()` and `top->in()` don't display their types, so can you please also print the type argument of the dtype view op?

Reply:
Added